build.gradle.kts

```diff
@@ -175,15 +180,33 @@ publishing {
     }
 }
 
-signing {
-    setRequired({
-        gradle.taskGraph.hasTask("publish")
-    })
-    sign(publishing.publications["mavenJava"])
+    repositories {
+        maven {
+            url = uri("$artifactoryContextUrl/libs-release-local")
+            credentials {
+                username = artifactoryUser
+                password = artifactoryPassword
+            }
+        }
+    }
 }
 
-nexusPublishing {
-    repositories {
-        sonatype()
+artifactory {
+    setContextUrl(artifactoryContextUrl)
+    publish {
+        repository {
+            setRepoKey("libs-release-local")
+            setUsername(artifactoryUser)
+            setPassword(artifactoryPassword)
+        }
+        defaults {
+            publications("mavenJava")
+            setPublishArtifacts(true)
+            setPublishPom(true)
+        }
     }
 }
+
+tasks.register("publishToArtifactory") {
+    dependsOn("publish")
+}
```

Comment:
Did the `artifactoryPublish` task not work?

Reply:
not mandatory, removed it.